package.json

```diff
@@ -1,26 +1,27 @@
 {
-    "name": "SmartSyncExplorerReactNative",
-    "version": "5.1.0",
-    "private": true,
-    "scripts": {
-        "start": "node node_modules/react-native/local-cli/cli.js start",
-        "test": "jest"
-    },
-    "dependencies": {
-        "react": "15.3.2",
-        "react-native": "0.35.0",
-        "salesforcemobilesdk-ios": "https://github.com/forcedotcom/SalesforceMobileSDK-iOS.git#dev",
-        "react-native-force": "https://github.com/forcedotcom/SalesforceMobileSDK-ReactNative.git#dev",
-        "salesforcemobilesdk-android": "https://github.com/forcedotcom/SalesforceMobileSDK-Android.git#dev"
-    },
-    "jest": {
-        "preset": "jest-react-native"
-    },
-    "devDependencies": {
-        "babel-jest": "16.0.0",
-        "babel-preset-react-native": "1.9.0",
-        "jest": "16.0.2",
-        "jest-react-native": "16.0.0",
-        "react-test-renderer": "15.3.2"
-    }
+  "name": "SmartSyncExplorerReactNative",
+  "version": "5.1.0",
+  "private": true,
+  "scripts": {
+    "start": "node node_modules/react-native/local-cli/cli.js start",
+    "test": "jest"
+  },
+  "dependencies": {
+    "react": "15.3.2",
+    "react-native": "0.35.0",
+    "react-native-force": "https://github.com/forcedotcom/SalesforceMobileSDK-ReactNative.git#dev",
+    "salesforcemobilesdk-android": "https://github.com/forcedotcom/SalesforceMobileSDK-Android.git#dev",
+    "salesforcemobilesdk-ios": "https://github.com/forcedotcom/SalesforceMobileSDK-iOS.git#dev",
+    "whatwg-fetch": "^1.1.1"
```

Comment:
I had to add `whatwg-fetch` explicitly here - otherwise it would fail. @wmathurin Have you run into this?